docs/UserGuide.md

```diff
@@ -160,22 +160,117 @@ Expected Output:
 Please provide a valid index of the course plan you would like to delete.
 ```
 
+### List out all the possible schools from the options: `list schools`
+List out all the schools users could possibly go for their SEP
+
+Format: `list schools`
+
+Expected Output:
+```
+-----------------------------------------------------
+The University of Melbourne
+The Australian National University
+Victoria University of Wellington
+The University of Western Australia
+-----------------------------------------------------
+```
+
+### Obtain contacts from the list of universities `obtain`
+Delete a course mapping plan that was initially saved into the Personal Tracker.
```

Comment:
I think this part was not updated for the overview for the `obtain` command!

Reply:
Okay, ammended! @chewycj